File: Backend/BackendModelLibrary/Model/NetworkDeviceDto.cs
namespace BackendModelLibrary.Model
{
    public class NetworkDeviceDto
    {
        public int NetworkDeviceID { get; set; }

        public int NetworkID { get; set; }

        public int DeviceID { get; set; }


        public NetworkDto? Network { get; set; }

        public DeviceDto? Device { get; set; }
    }


}

File: frontend/src/Models/NetworkDeviceDto.ts
import DeviceDto from "./DeviceDto";
import NetworkDto from "./NetworkDto";

export default interface NetworkDeviceDto {
    NetworkDeviceID: number;
    NetworkID: number;
    DeviceID: number;
    NetworkDto: NetworkDto | undefined | null;
    DeviceDto: DeviceDto | undefined | null
}